int32 id1
int32 id2
string type1
string type2
string[] relations
---
semap_msgs/LabeledGeometryCollection[] results

